homeassistant/components/overkiz/diagnostics.py

```diff
@@ -0,0 +1,21 @@
+"""Provides diagnostics for Overkiz."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from . import HomeAssistantOverkizData
+from .const import DOMAIN
+
+
+async def async_get_config_entry_diagnostics(
+    hass: HomeAssistant, entry: ConfigEntry
+) -> list[dict[str, Any]]:
+    """Return diagnostics for a config entry."""
+    data: HomeAssistantOverkizData = hass.data[DOMAIN][entry.entry_id]
+    client = data.coordinator.client
+    setup = await client.get_diagnostic_data()
+
+    return [setup]
```

Comment:
why make it a list?

Reply:
... I was almost sure that the return format was a list, thus that's why I formatted it as a list. However I see now that it should be a dict, thus that my list conversion is not required at all. Thanks!